lambdas/src/apis/collections/controllers/wearables.ts

```diff
@@ -113,7 +113,7 @@ export async function getWearablesEndpoint(
 
     res.send({ wearables, filters: requestFilters, pagination: { limit: sanitizedLimit, lastId, next } })
   } catch (error) {
-    res.status(500).send(error.message)
+    res.status(error.code ?? 500).send(error.message)
```

Comment:
As a rule of thumb, in any service, we should _never_ forward the errors to the clients. Only logging them to the logger. https://owasp.org/www-community/Improper_Error_Handling

Reply:
Edit: I was suggesting to remove the `.send(error.message)`